packages/web-frontend/src/containers/Map/Map.js

```diff
@@ -75,10 +75,6 @@ class MapComponent extends Component {
     } = this.props;
     // Only updates/re-renders when the measure has changed or the orgUnit has changed.
     // These are the only cases where polygons or area tooltips should rerender.
-    if (nextProps.measureInfo.measureId !== measureInfo.measureId) {
-      return true;
-    }
-
```

Comment:
That's a good point @kael89 , I feel that is a trade off because `measureId` is not being used in this component. However I'm happy to put it back for future usage, I guess it can be used for another map overlay refactoring.